### ISSUE TITLE
Problem on start DisableWinTracking version 3.1.1 and 3.1.2

### ISSUE THREAD

**Issue by meshkov** [10se1ucgo/DisableWinTracking#359]
**INFORMATION:**

* Windows Edition (Home/Pro/Enterprise): Windows 10 Pro (1703, 15063.632)
* Architecture (32bit/64bit): 64bit
* Options used: -

Error on DisableWinTracking version 3.1.1 and 3.1.2:
![image](https://user-images.githubusercontent.com/1703670/30886331-4de396be-a31f-11e7-965b-c14abfd10634.png)

**LOG:**

```python
00:57:11 INFO: Python 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:42:59) [MSC v.1500 32 bit (Intel)] on win32
00:57:11 INFO: ('Windows', 'DESKTOP-B9NURL6', '10', '10.0.15063', 'AMD64', 'Intel64 Family 6 Model 58 Stepping 9, GenuineIntel')
00:57:11 INFO: DisableWinTracking version 3.1.1
00:57:12 CRITICAL: Traceback (most recent call last):
  File "dwt.py", line 486, in <module>
  File "dwt_about.py", line 172, in update_check
NameError: global name 'URLError' is not defined
```

**Comment by Biswa96** [comment]
Did you disable NIC when you ran this app?

**Comment by AkatsukiNin** [comment]
I got the same error on Win10 Pro Version 1607 OS build 14393.1715

**Comment by meshkov** [comment]
@Biswa96 no, i don`t disable NIC

**Comment by yaseenshaik** [comment]
I got this error too. Turns out you need to have internet access and an open firewall for this app to work. Huh, the irony....

**Comment by barul42** [comment]
I also got the error, but when I tried to run the script from source I didn't have any issue; might be worth a try.

**Comment by Ruined1** [comment]
The reason this crash is happening (I believe) is due to a lack of an exception check if the program is unable to see if an update is available.

**Comment by Ruined1** [comment]
@meshkov @yaseenshaik it's now fixed in v3.1.3, no longer need to have an internet connection to run as the error checking is now handled properly.
